specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/apimdeployment.json

```diff
@@ -0,0 +1,2113 @@
+            "values": [
+              {
+                "value": "None",
+                "description": "The service is not part of any Virtual Network."
+              },
+              {
+                "value": "External",
+                "description": "The service is part of Virtual Network and it is accessible from Internet."
+              },
+              {
+                "value": "Internal",
+                "description": "The service is part of Virtual Network and it is only accessible from within the virtual network."
+              }
+            ]
+          }
+        },
+        "apiVersionConstraint": {
+          "$ref": "#/definitions/ApiVersionConstraint",
+          "description": "Control Plane Apis version constraint for the API Management service."
+        },
+        "restore": {
+          "type": "boolean",
+          "description": "Undelete Api Management Service if it was previously soft-deleted. If this flag is specified and set to True all other properties will be ignored.",
+          "default": false
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "items": {
+            "$ref": "./definitions.json#/definitions/RemotePrivateEndpointConnectionWrapper"
+          },
+          "description": "List of Private Endpoint Connections of this service."
+        },
+        "platformVersion": {
+          "type": "string",
+          "description": "Compute Platform Version running the service in this location.",
+          "readOnly": true,
+          "enum": [
+            "undetermined",
+            "stv1",
+            "stv2",
+            "mtv1",
+            "stv2.1"
+          ],
+          "x-ms-enum": {
+            "name": "PlatformVersion",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "undetermined",
+                "description": "Platform version cannot be determined, as compute platform is not deployed."
+              },
+              {
+                "value": "stv1",
+                "description": "Platform running the service on Single Tenant V1 platform."
+              },
+              {
+                "value": "stv2",
+                "description": "Platform running the service on Single Tenant V2 platform."
+              },
+              {
+                "value": "mtv1",
+                "description": "Platform running the service on Multi Tenant V1 platform."
+              },
+              {
+                "value": "stv2.1",
+                "description": "Platform running the service on Single Tenant V2 platform on newer Hardware."
+              }
+            ]
+          }
+        },
+        "legacyPortalEnabled": {
+          "type": "boolean",
+          "description": "Enable legacy portal for this API Management service.",
+          "default": true
+        },
+        "developerPortalEnabled": {
+          "type": "boolean",
```

Comment:
>boolean [](http://example.com/codeflow?start=19&length=7)

consider using enum instead:
[https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md#why-the-rule-is-important](https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md#why-the-rule-is-important)